src/elements/hx-switch-control/index.js

```diff
@@ -0,0 +1,27 @@
+import { HXFormControlElement } from '../../interfaces/HXFormControlElement/index.js';
```

Comment:
@lalithkarikelli, I modified the `Switch` component and added `<hx-switch-control>`.   We'll discuss these updates.